src/Parser/GeoJsonParser.js

```diff
@@ -51,6 +51,15 @@ const toFeature = {
         geometry.properties = properties;
         geometry.properties.style = new Style().setFromGeojsonProperties(properties, feature.type);
         this.populateGeometry(crsIn, coordsIn, geometry, setAltitude, feature);
+
+        // TODO: take a decision for labels with other FEATURE_TYPES
```

Comment:
We should keep `FEATURE_TYPES.POINT` and have a convert `FeaturePoint2Label`

Reply:
I moved the parts in parsers to `LabelLayer#convert`.